scripts/addons/io_scene_gltf2/gltf2_generate.py

```diff
@@ -1523,27 +1523,34 @@ def generate_meshes(operator,
 
         #
 
+        if export_settings['gltf_extras']:
+            extras = create_custom_property(blender_mesh)
+
+            if extras is not None:
+                mesh['extras'] = extras
+
+        #
+
         if export_settings['gltf_morph']:
             if blender_mesh.shape_keys is not None:
                 morph_max = len(blender_mesh.shape_keys.key_blocks) - 1
                 if morph_max > 0:
                     weights = []
+                    targetNames = []
```

Comment:
Could you change the variable name to `target_names` here for consistency? (But exporting `mesh['extras']['targetNames']` i agree with).